const url = '';

fetch(url)
  .then(function(res){
  })
  .then(function(json){
  })
  .then(function(data){
  })

  .catch(function(err){
    //handles error for all the function no need of handling error in each of the function as callbaack hell
  })
